source/geometries/Next100SiPM.cc

```diff
@@ -71,6 +74,33 @@ void Next100SiPM::Construct()
 
   GeometryBase::SetLogicalVolume(sipm_logic_vol);
 
+  // COATING /////////////////////////////////////////////
+  if (coating_thickn_ > 0.) {
+    G4LogicalVolume* coating_logic_vol;
+
+    G4String coating_name = sipm_name + "_WLS";
+
+    G4Box* coating_solid_vol =
+      new G4Box(coating_name, sipm_width/2., sipm_length/2., coating_thickn_/2.);
+
+    G4Material* coating_mt = MaterialsList::TPB();
+    coating_mt->SetMaterialPropertiesTable(OpticalMaterialProperties::TPB());
+
+    coating_logic_vol = new G4LogicalVolume(coating_solid_vol, coating_mt, coating_name);
```

Comment:
I meant to `G4LogicalVolume* coating_logic_vol = new blah blah`

Reply:
Changed it!